### ISSUE TITLE
Release transport implementation in the rubygems

### ISSUE THREAD

**Issue by repeatedly** [awslabs/logstash-output-amazon_es#1]
Do you have a plan to release transport layer as a library?
I'm a maintainer of Fluentd and I received many questions `When fluentd supports AES?`.
If you release transport layer as a library, we can easy to support AES with existing elasticsearch plugin.

If we have an other better way, please let me know. Thanks.

**Comment by malpani** [comment]
@repeatedly : Yes, this makes sense and might be something lot of people would benefit from. Will post an update on when we can get this out.

**Comment by repeatedly** [comment]
@malpani Cool! I can't wait it!
